patches/server/0020-Monumenta-Fix-shulker-NoAI-allowing-peeking.patch

```diff
@@ -0,0 +1,38 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ziggleflig <dmdinnc@gmail.com>
+Date: Sun, 1 May 2022 00:46:53 -0400
+Subject: [PATCH] Monumenta - Fix shulker NoAI allowing peeking
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/monster/Shulker.java b/src/main/java/net/minecraft/world/entity/monster/Shulker.java
+index a9dfe190f46230077e2e1bf9aacbf5375651f216..1077cc08b0f716300d7c278bb345a25d72cb1cbe 100644
+--- a/src/main/java/net/minecraft/world/entity/monster/Shulker.java
++++ b/src/main/java/net/minecraft/world/entity/monster/Shulker.java
+@@ -94,6 +94,7 @@ public class Shulker extends AbstractGolem implements Enemy {
+ 
+     @Override
+     protected void registerGoals() {
++        if (this.isNoAi()) return;
```

Comment:
Wouldn't this prevent a shulker from functioning if its AI were later enabled? I think we have some abilities that apply NoAI, so this could make cheesing shulkers easier for, say, LG.